pkg/provider/pods.go

```diff
@@ -202,3 +202,13 @@ func (p *Pod) CreatedByDeploymentConfig() (bool, error) {
 	}
 	return false, nil
 }
+
+func (p *Pod) HasNodeAssignment() bool {
```

Comment:
Is it not the same?

`return p.Spec.NodeSelector != nil || p.Spec.NodeName != ""`

Reply:
You're right!  I'll adjust.